.github/workflows/migration.yml

```diff
@@ -34,13 +37,15 @@ jobs:
       with:
         node-version: '12'
```

Comment:
We should probably update this to use node 20, which may require updating the rest of the actions in this file, but will help us ensure we are on the latest, most secure, version. This can be done in a separate PR.